src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -43,6 +43,26 @@ const QString kPdbPath = QStringLiteral("PIONEER/rekordbox/export.pdb");
 const QString kPLaylistPathDelimiter = QStringLiteral("-->");
 const double kLongestPosition = 999999999.0;
 
+enum TrackColor {
```

Comment:
Please use an `enum class` and make the members `PascalCase`

Reply:
Slipped through. The constants above this definition could become `constexpr`.

Reply:
Attempted to change to `constexpr` however seem to be getting compiler errors each way I try:
`src/library/rekordbox/rekordboxfeature.cpp:43:19: error: constexpr variable cannot have non-literal type 'const QString'
constexpr QString kRekordboxLibraryTable = QStringLiteral("rekordbox_library");`

`/usr/local/Cellar/qt/5.12.2/Frameworks/QtCore.framework/Headers/qstring.h:1135:17: note: 'QString' is not literal because it has a user-provided destructor
inline QString::~QString() { if (!d->ref.deref()) Data::deallocate(d); }`

Reply:
QString does not work as constexpr, only double